.github/workflows/lexbox-api.yaml

```diff
@@ -50,8 +50,15 @@ jobs:
       - uses: actions/setup-dotnet@v4
         with:
           dotnet-version: '8.x'
+      - name: Install Task
+        uses: arduino/setup-task@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Dotnet build
         run: dotnet build LexBoxOnly.slnf
+      - name: Check for pending EF model changes
+        run: task api:has-pending-model-changes
```

Comment:
Apparently you can do this in C# as well: `context.Database.HasPendingModelChanges()`. Do you prefer this?

It's not obvious to me from the documentation that this command would interrupt a GHA workflow. But, I'm assuming you know what you're doing.

Reply:
yes this will return a non zero exit code failing the build.

Oh that's interesting, I'm not sure I want to prevent the app from starting up as that could be pretty frustrating.